### PR TITLE
Update fonts and refresh locks

### PR DESCRIPTION
**Pull request by barakj**
This PR addresses: 

1. Updating fonts to new square sans fonts.

2. Fixing inconsistencies and bugs I've found, and refreshing lock files for node related apps.

3. The gift card sample app was using the cards API to create customer card on file, but this API changed a little (lag between creation and listing, no card brand information for test nonce). I had to make code modifications so that the app doesn't crash.

4. Missing gitgnores for `.env` files in some samples

### CODE REVIEW

File: connect-examples/v2/node_gift-cards/views/functions.ejs [hunk 1]
```diff
@@ -177,13 +177,16 @@ getCustomerName = function(firstName, lastName, length = 40) {
...
+  return cards.map(card => {
+    card.cardBrand = getCardBrand(card);
+    return {
+      img: getImageForBrand(card.cardBrand),
+      value: card.id,
+      displayValue: capitalize(card.cardBrand) + " ●●●● " + card.last4,
+      description: capitalize(card.cardBrand)
+    }
+  });
+  return test;
```
Comment (azlee) [inline connect-examples/v2/node_gift-cards/views/functions.ejs:189]: I think this line can be removed

File: connect-examples/v2/node_invoices/public/stylesheets/index.css [hunk 2]
```diff
@@ -14,6 +14,24 @@ See the License for the specific language governing permissions and
...
+@import url(https://square-fonts-production-f.squarecdn.com/square-sans.min.css);
+
+html, body {
+    color: var(--dark-gray);
+    font-size: 16px;
+    line-height: 24px;
+    font-family: var(--square-sans-text);
+    font-weight: normal;
+    font-style: normal;
+    color:  #151C1F;
```
Comment (azlee) [inline connect-examples/v2/node_invoices/public/stylesheets/index.css:26]: This line can be removed

File: connect-examples/v2/node_invoices/public/stylesheets/invoice.css [hunk 3]
```diff
@@ -13,6 +13,19 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+@import url(https://square-fonts-production-f.squarecdn.com/square-sans.min.css);
+
+html, body {
+    color: var(--dark-gray);
+    font-size: 16px;
+    line-height: 24px;
+    font-family: var(--square-sans-text);
+    font-weight: normal;
+    font-style: normal;
+    color:  #151C1F;
```
Comment (azlee) [inline connect-examples/v2/node_invoices/public/stylesheets/invoice.css:25]: This line can be removed

**Review by azlee — APPROVED** [review]

File: connect-examples/v2/node_gift-cards/views/functions.ejs (hunk 1 above)
Comment (barakj) [reply connect-examples/v2/node_gift-cards/views/functions.ejs:189]: good eye, forgot to remove this

**Pull request merged**